.snyk

```diff
@@ -0,0 +1,4 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
What is this file?

Reply:
This file is a file generated by a tool used during the analyses. I'm going to delete it.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad